package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "update-pr-branch",
-  "version": "0.1.2",
+  "version": "0.9.0",
```

Comment:
What versioning system is this using to go from 0.1.2 to 0.9.0?

Reply:
removed, thanks @bafolts